src/app/CommandHandler.h

```diff
@@ -363,6 +368,11 @@ class CommandHandler : public Messaging::ExchangeDelegate
      */
     CHIP_ERROR AllocateBuffer();
 
+    CHIP_ERROR ValidateCommands(TLV::TLVReader & invokeRequestsReader);
+
+    CHIP_ERROR PrepareCommand(const CommandRefLookupTable::CommandRefTableEntry * apCommandRefTableEntry,
```

Comment:
Please document this version, perhaps rename it as well, since htere are public methods of similar name.